coralproject/talk
# [CORL-378] AMP Support

## What does this PR do?

- Adds AMP support

## How do I test this PR?
- In Development you'll see another story page called `AMP`
- You can embed inside amp using an url pattern like `http://mycoralserver.com/embed/stream/amp?storyURL=http://story.url` or instead `?storyURL=` you can also use `?storyID=`
